Sources/Networking/AlamofireNetworkClient/OAuth/OAuthInterceptor.swift

```diff
@@ -13,19 +13,23 @@ import PovioKit
 public struct OAuthContainer {
   public let accessToken: String
   public let refreshToken: String
+  public let expirationDate: Date
```

Comment:
Should this be optional? What if we don't have that mechanism defined?

Reply:
Applied some changes, please take a look again.